libs/resolver/deno_json.rs

```diff
@@ -662,7 +680,9 @@ impl CompilerOptionsResolver {
     let mut ts_config_collector =
       TsConfigCollector::new(sys, node_resolver, &logged_warnings);
     for (dir_url, dir) in workspace_directory_provider.entries() {
-      ts_config_collector.add_root(dir.dir_path().join("tsconfig.json"));
+      if dir.has_deno_or_pkg_json() {
```

Comment:
One thing I just realized is I think we really just have to do this because `dir.has_deno_or_pkg_json()` will always be false when `config_discover == ConfigDiscoverOption::Disabled`.